src/app/databaseService.ts

```diff
@@ -237,13 +231,27 @@ async function getEvlFeedByVrm(
   return [result];
 }
 
-async function getEvlFeed(
+function selectQuery(feedName: FeedName) {
```

Comment:
Instead of this selectQuery function we could potentially have a map like this
```suggestion
  const getQueryMap: ([key: FeedName]: string) {
  EVL: EVL_QUERY,
  TFL: TFL_QUERY
  }
  ```

Then in `getFeed` you can use `getQueryMap[feedName]` then do an undefined check and throw error if it couldn't find it. Just another way in order to reduce if statement chaining if we think there'll be more. Your call, I just did something similar in the authoriser to avoid a bunch of switch cases statements

Reply:
I like this way more, I'll add this post meeting.